### PR TITLE
Next.js: Use NPM package of JavaScript SDK

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added steps for using NPM package of JavaScript SDK in Next.js app
https://www.notion.so/rudderstacks/Doc-update-for-next-js-app-using-npm-package-of-the-JS-SDK-86814f731c714fb4a715e6cb8000a1e9

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx [hunk 1]
```diff
@@ -100,7 +100,56 @@ export default function Home() {
 
 <div class="infoBlock">
 
-Refer to the sample Next.js app in the <a href="https://github.com/rudderlabs/rudder-samples">RudderStack Sample Applications</a> repository for more information.
+Refer to the sample Next.js app in the [RudderStack Sample Applications](https://github.com/rudderlabs/rudder-samples) repository for more information.
```
Comment (saikumarrs) [inline docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx:103]: Let's point to the actual example in the repository.

File: docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx [hunk 2]
```diff
@@ -100,7 +100,56 @@ export default function Home() {
...
+
+  const search = () => {
+    window.rudderanalytics.page();
+    window.rudderanalytics.track("Track Event.");
+  };
+```
+
+<div class="infoBlock">
+
+Refer to the sample Next.js app in the [RudderStack Sample Applications](https://github.com/rudderlabs/rudder-samples) repository for more information.
```
Comment (saikumarrs) [inline docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx:151]: Let's point to the actual example in the repository.

File: docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx [hunk 3]
```diff
@@ -100,7 +100,56 @@ export default function Home() {
...
+import { rudderInitialize } from "./rudderInitialize";
+
+export default function Home() {
+  React.useEffect(() => {
+    rudderInitialize();
+  }, []);
+
+  const search = () => {
+    window.rudderanalytics.page();
+    window.rudderanalytics.track("Track Event.");
```
Comment (saikumarrs) [inline docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx:145]:
```suggestion
    window.rudderanalytics.track("Sample Track Event");
```

File: docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx [hunk 4]
```diff
@@ -100,7 +100,56 @@ export default function Home() {
...
+
+  const search = () => {
+    window.rudderanalytics.page();
+    window.rudderanalytics.track("Sample Track Event");
+  };
+```
+
+<div class="infoBlock">
+
+Refer to the sample Next.js app in the [RudderStack Sample Applications](https://github.com/rudderlabs/rudder-samples/tree/master/javascript/sample-next-app) repository for more information.
```
Comment (MoumitaM) [inline docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx:151]: Sample project with NPM package can be find here: https://github.com/rudderlabs/rudder-analytics-next/tree/main/sample-next-app-npm
Not in sample apps

**Review by MoumitaM — APPROVED** [review]

**Review by ameypv-rudder — APPROVED** [review]

File: docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx [hunk 5]
```diff
@@ -100,7 +100,55 @@ export default function Home() {
...
+
+  const search = () => {
+    window.rudderanalytics.page();
+    window.rudderanalytics.track("Sample Track Event");
+  };
+```
+
+<div class="infoBlock">
+
+Refer to the sample Next.js app in the <a href="https://github.com/rudderlabs/rudder-samples/tree/master/javascript/sample-next-app">RudderStack Sample Applications</a> repository for more information.
```
Comment (ameypv-rudder) [inline docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx:151]:
```suggestion
Refer to the sample Next.js app in the [RudderStack Sample Applications](https://github.com/rudderlabs/rudder-analytics-next/tree/main/sample-next-app-npm) repository for more information.
```

File: docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx [hunk 6]
```diff
@@ -100,7 +100,55 @@ export default function Home() {
 
 <div class="infoBlock">
 
-Refer to the sample Next.js app in the <a href="https://github.com/rudderlabs/rudder-samples">RudderStack Sample Applications</a> repository for more information.
+Refer to the sample Next.js app in the <a href="https://github.com/rudderlabs/rudder-samples/tree/master/javascript/sample-next-app">RudderStack Sample Applications</a> repository for more information.
```
Comment (ameypv-rudder) [inline docs/user-guides/how-to-guides/rudderstack-jamstack-integration/rudderstack-nextjs-integration.mdx:103]:
```suggestion
Refer to the sample Next.js app in the [RudderStack Sample Applications](https://github.com/rudderlabs/rudder-analytics-next/tree/main/sample-next-app-npm) repository for more information.
```

**Pull request merged**